Makefile

```diff
@@ -1,11 +1,10 @@
 GOTOOLS = \
-	github.com/golang/dep/cmd/dep \
-	# gopkg.in/alecthomas/gometalinter.v2
+	github.com/golang/dep/cmd/dep
 PACKAGES=$(shell go list ./... | grep -v '/vendor/')
 BUILD_TAGS?=tendermint
 BUILD_FLAGS = -ldflags "-X github.com/tendermint/tendermint/version.GitCommit=`git rev-parse --short=8 HEAD`"
 
-all: check build test install
+all: check build test_integrations install
```

Comment:
do we really want to run test_integrations here?

Reply:
IIRC this was requested (somewhere, can't remember)